cms/admin/permissionadmin.py

```diff
@@ -51,8 +51,9 @@ def get_queryset(self, request):
         # can see only permissions for users which are under him in tree
 
         # here an exception can be thrown
+        model = self.model
```

Comment:
Why using a different code style instead of doin ``qs = self.model...`` like for PageAdmin?